### DISCUSSION TITLE
Text input disabled when Unity app is rendered

### DISCUSSION THREAD

**Discussion by Oba-One** [jeffreylanters/react-unity-webgl#485] [Help Wanted!]
Hi! I'm running into a weird bug where when the Unity game is loaded in the app the react components I have overlaying with a chat feature for people to send messages during the game isn't working when inputting text. I can paste text but not type.

**Comment by jeffreylanters** [comment]
Hi! Please see the Tab Index API for more information on how to to prevent Unity for capturing all the input on the webpage. 

https://react-unity-webgl.dev/docs/api/tab-index
